ottreeindex/scripts/README.md

```diff
@@ -3,18 +3,73 @@ This directory contains scripts for setting up the database from the command lin
 # Files:
 
 For command-line setup (for testing not from within pyramid):
+
 * config.yml: YAML file with database settings, including table names.
 * setup_db.py: creates DB tables if do not already exist; optionally deleting them all first
 * load_nexson.py: loads the nexson files into the database and creates the JSON index
-* create_otu_table.py: generates a csv file of otu-tree relationships and then loads this using the COPY command
+* create\_otu\_table.py: generates a csv file of otu-tree relationships and then loads this using the COPY command
+* initializedb.py: the pyramid file for initializing the database
```

Comment:
initializedb.py should not be in this list, i.e. under the heading "command line setup (for testing not from within pyramid)"